subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -132,7 +132,6 @@ config BT_GATT_CACHING
 	bool "GATT Caching support"
 	default y
 	depends on BT_GATT_SERVICE_CHANGED
-	depends on PSA_CRYPTO_CLIENT
```

Comment:
I think this is depends on is required, dependency loop can be resolved by removing BT_GATT_CACHING from here https://github.com/nrfconnect/sdk-nrf/pull/19720/commits/c72d3f8789a380c15494324d8f104176c153035f#diff-4e208770b414cbc5f054590c4b31de82d1ed30611d8b392a59568644943b7d74R124 - unless I am missing something